### ISSUE TITLE
Can we receive Telegram Group Messages/images in WhatsApp Groups?

### ISSUE THREAD

**Issue by capsload3** [akshettrj/watgbridge#49]
Hey,

is it possible to forward Telegram Group Messages/Images to WhatsApp Groups?

**Comment by akshettrj** [comment]
No, this project is more geared towards personal usage and not for creating a bridge between a particular Telegram and a WhatsApp group
